flask_security/forms.py

```diff
@@ -203,8 +204,9 @@ def validate(self):
 class LoginForm(Form, NextFormMixin):
```

Comment:
I am not sure why we can't use `EmailFormMixin` and `PasswordFormMixin` here too.